### ISSUE TITLE
Fast Tracker feat missing Agility prerequisite in Open Legend site

### ISSUE THREAD

**Issue by jcte02** [openlegend/core-rules#106]
In the Open Legend site the [Fast Tracker](http://www.openlegendrpg.com/feats/fast_tracker) feat prerequisites are shown as: 

**Prerequisites**
* **Tier 1:** Perception 5

But in the feats.yml file the prerequisites are the following:
```yaml
prerequisites:
  tier1:
    Attribute:
      - Agility: 5
    Attribute:
      - Perception: 5
```
and from what I understand the site should render it as:

**Prerequisites**
* **Tier 1:**
  * Agility 5
  * Perception 5

I think the problem is in the yaml file since this is the result of the conversion to json (the format that the site use from my understandings):

```json
{
  "prerequisites": {
    "tier1": {
      "Attribute": [
        {
          "Perception": 5
        }
      ]
    }
  }
}
```

which is compatible with the erroneus prerequisites shown on the site.

**Comment by raihje** [comment]
There are a few, Fleet of Foot and Multi-Target Boon Expert also don't render properly.

**Comment by brianfeister** [comment]
Ugh! This can't produce valid JSON. Keys are unique... Hmmmm

```
prerequisites:
  tier1:
    Attribute:
      - Agility: 5
    Attribute:
      - Perception: 5
```

**Comment by jcte02** [comment]
A simple fix would be to squash it into a multi-array like:
```yaml
prerequisites:
  tier1:
    Attribute:
      -
        - Agility: 5
      -
        - Perception: 5
```
that produces the following json result:
```json
{
  "prerequisites": {
    "tier1": {
      "Attribute": [
        [
          {
            "Agility": 5
          }
        ],
        [
          {
            "Perception": 5
          }
        ]
      ]
    }
  }
}
```
which is not perfect but manageable.
A more radical approach is to restructure the Attribute prerequisite like:
```yaml
Attribute:
  - Any:
    # you need to have at least one of these attributes eg. for Battlefield Defender
    - Agility: 4
    - Might: 4
    - Protection: 4
  - All:
    # all attributes in this list are requires e.g. for Fast Tracker
    - Agility: 5
    - Perception: 5
```
which requires to rewrite the entire feats file (a long and error prone process).

**Comment by brianfeister** [comment]
@jcte02 thanks for the inspiration, it jarred my thinking enough that I found something suitable, this is the standard now:

```yaml
- !
  name: Fast Tracker
  prerequisites:
    tier1:
      - Attribute:
        - Agility: 5
      - Attribute:
        - Perception: 5
```

**Comment by jcte02** [comment]
@brianfeister 
I personally think that breaking the code style (i.e. introducing a new style just for multiple attributes) is not a very optimal solution since it can lead to inconsistencies.
While thinking I came up with this solution:
```yaml
prerequisites:
  tier1:
    MultiAttribute:
      - Agility: 5
      - Perception: 5
```
which follows the current style i.e. introducing a new key for particular cases (e.g. FeatAttributes) and also maintain consistency with the other feats.

**Comment by jcte02** [comment]
This is a little off-topic, but when listing the attributes in feat prequisites you can do something like
```yaml
prerequisites:
  tier1:
    MultiAttribute:
      Agility: 5
      Perception: 5
```
which yields
```json
{
  "prerequisites": {
    "tier1": {
      "MultiAttribute": {
        "Agility": 5,
        "Perception": 5
      }
    }
  }
}
```
instead of
```yaml
prerequisites:
  tier1:
    MultiAttribute:
      - Agility: 5
      - Perception: 5
```
that results in
```json
{
  "prerequisites": {
    "tier1": {
      "MultiAttribute": [
        {
          "Agility": 5
        },
        {
          "Perception": 5
        }
      ]
    }
  }
}
```
As you can see not listing them as array items, instead of an array containing partial hashes, we obtain a single hash, which I think is easier to parse.
I can do the conversion and make a pull request if you fancy this idea.

**Comment by brianfeister** [comment]
@jcte02 - great suggestion. Good thing I've got you here to keep me honest

**Comment by jcte02** [comment]
Closed by #148.
